### DISCUSSION TITLE
MEM: Add BW, St/mono & squelch

### DISCUSSION THREAD

**Discussion by AaronJowitt** [PE5PVB/TEF6686_ESP32#358] [Ideas]
A number of tuners store frequency, band, stereo/mono mode, mute status & BW in each preset/MEM location.  

Maybe desirable to add BW, St/mono & squelch value to MEM store & recall. Thanks.

**Comment by PE5PVB** [comment]
Squelch is not possible, because this is linked to the position of the rotary encoder. BW and stereo/mono can be done.
